reference/docs-conceptual/community/contributing/editorial-checklist.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Editorial checklist
+description: This is a summarized list of rules for editing PowerShell documentation.
+ms.date: 03/05/2020
+ms.topic: conceptual
+---
+# Editor's checklist
```

Comment:
Do we also want to specify that files should always be lowercased and no special characters in the file names? It may be somewhere, but I don't recall seeing it and we should call it out.

Reply:
meh